Lib/test/test_unicodedata.py

```diff
@@ -71,7 +71,7 @@ class UnicodeFunctionsTest(UnicodeDatabaseTest):
 
     # Update this if the database changes. Make sure to do a full rebuild
     # (e.g. 'make distclean && make') to get the correct checksum.
-    expectedchecksum = '98d602e1f69d5c5bb8a5910c40bbbad4e18e8370'
+    expectedchecksum = '4975f3ec0acd4a62465d18c9bf8519b1964181f6'
```

Comment:
Would you mind also adding a test that does some direct comparisons (like `assert unicodedata.east_asian_width(X) == Y` for some `Y`'s that used to be wrong).

Reply:
I've added the test back, and also added some cases for characters in private use areas (which need to return "A"), thanks for making me carefully look at the standard again ;-).